vars/gitUtils.groovy

```diff
@@ -0,0 +1,189 @@
+/**
```

Comment:
I would name it as `githubUtils.groovy` because the code here is not only for git.